QtQuickApplication/TabGeneral.qml

```diff
@@ -140,13 +144,102 @@ Rectangle{
                 anchors.top: rpmLabel.bottom
                 anchors.right: rpmLabel.right
             }
+
+            Row {
+                id: statusRow
+                anchors.bottom: generalInfoTile.bottom
+                anchors.left: parent.left
+                anchors.right: parent.right
+                height: parent.height * 0.25
+
+                Rectangle {
```

Comment:
you can put this rectangle into a custom qml type, because you use it multiple times